language-design.md

```diff
@@ -58,8 +134,20 @@ In what way is an API a language?
 What does the post on grayscale tell us about the process of API design?
 
 **Response**
-
-
+The greyscale post offers four options for functions that produce shades of grey. At
+the end, it includes a poll that asks readers to vote on the most intuitive one
+and also encourages users to leave a comment justifying their opinions [Verou, 2014].
+This shows that API design is very influenced by user feedback and is very concerned
+with how intuitive their functions are. Although some of the comments are not
+constructive, the comments section allows for a large set of users to give 
+quick feedback. In this way, a blog post is the best medium to help the API design
+process because it allows for a large user base to give feedback, hopefully
+leading to improvements in the API.
+
+This particular example also emphasizes how there might not be a correct answer to some 
+problems. Clearly there are many differing views in the comments, and even the final result 
+might not "feel right" to some users. But at least there is some documentation explaining why 
+someone would name the greyscale in a particular way.
```

Comment:
I agree there might not be a right answer. The right answer, as you might believe, is dependent on the users. Listening to users and their needs is a good way to get a consensus/approach.

It was cool that there is some documentation explaining why someone would name the grayscale in a particular way!